varia/Dockerfile.tests

```diff
@@ -8,7 +8,14 @@ ENV RUSTFLAGS="-D warnings"
 RUN apt-get update && \
     apt-get install -y libacl1-dev && \
     rm -rf /var/lib/apt/lists/*
+
+# Build as unprivileged user
+RUN useradd build --create-home
+WORKDIR /home/build
+USER build
```

Comment:
Use unprivileged user: tests are relying on the fact that `/root` should not be accessible (permission denied)